package/MDAnalysis/core/universe.py

```diff
@@ -247,7 +255,14 @@ def __init__(self, *args, **kwargs):
                 parser = get_parser_for(self.filename, format=topology_format)
                 try:
                     with parser(self.filename) as p:
-                        self._topology = p.parse()
+                        if parser is _PARSERS['MINIMAL']:
+                            # The MINIMAL parser just creates a n_atoms
+                            # topology. Providing kwargs with n_atoms allows
+                            # shortcutting and handling coordinate/trajectory
+                            # files not aware of their own atom count.
+                            self._topology = p.parse(**kwargs)
```

Comment:
ie in all topology parsers, not just minimal.  I'd prefer to keep the API consistent by adding this to all parsers